firefox-ios/Client/Frontend/Autofill/Address/Edit/AddressModifiedStatus.swift

```diff
@@ -5,15 +5,42 @@
 import Foundation
 
 enum AddressModifiedStatus {
+    enum ErrorType {
+        case save(action: () -> Void)
+        case update(action: () -> Void)
+        case remove(action: () -> Void)
+
+        var message: String {
+            switch self {
+            case .save: return .Addresses.Settings.Edit.AddressSaveError
+            case .update: return .Addresses.Settings.Edit.AddressSaveError
+            case .remove: return .Addresses.Settings.Edit.AddressRemoveError
```

Comment:
Do they all have same error? 🤔

Reply:
Only the saving and updating.

Reply:
@razvanlitianu debating if we should combine this but R+ regardless 

```suggestion
            case .save, .update: return .Addresses.Settings.Edit.AddressSaveError
            case .remove: return .Addresses.Settings.Edit.AddressRemoveError
```